test-clients/src/main/java/com/hedera/services/bdd/suites/misc/GuidedTourRemoteSuite.java

```diff
@@ -76,14 +71,67 @@ private List<HapiApiSpec> guidedTour() {
                 balanceLookupContractWorks());
     }
 
+    /**
+     * Provides an example of a spec that replaces a target account's key with a 1/2 threshold key,
+     * where the two top-level keys are the target's existing Ed25519 key and a 2/3 threshold key
+     * with all these three keys simple Ed25519.
+     *
+     * <p>All the keys are imported from PEM files.
+     *
+     * @return the example spec
+     */
+    private HapiApiSpec rekeyAccountWith2Of3Choice() {
```

Comment:
Is this not enabled to run ?

Reply:
No, I just put this as an example so that we copy+paste an example of how to use it.